index.html

```diff
@@ -1,21 +1,103 @@
+    />
+    <link
+      href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <script
+      src="https://kit.fontawesome.com/02e465af65.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <!-- Top Navigation Menu -->
+      <nav class="navbar">
+        <div class="content">
+          <div class="logo">
+            <a href="#"><img src="favicon.ico" alt="logo" /></a>
+          </div>
+          <ul class="menu-list">
+            <div class="icon cancel-btn">
+              <i class="fas fa-times"></i>
+            </div>
+            <li><a href="#">Meet Karma</a></li>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Blog</a></li>
+            <li><a href="#">Help</a></li>
+            <li><a href="#">Login</a></li>
+          </ul>
+        </div>
+        <div class="icon menu-btn">
+          <i class="fas fa-bars"></i>
+        </div>
+      </nav>
+    </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <main>
+      <section class="hero">
+        <div class="hero-content">
+          <h1>Introducing Karma</h1>
+          <p>Bring WiFi with you, everywhere you go.</p>
+          <button>Learn More</button>
+        </div>
+      </section>
+      <section class="section-devices">
+        <h1>Everyone needs a little Karma</h1>
+        <div class="container-devices">
+          <article class="devices">
+            <a>
+              <img src="img/icon-devices.svg" alt="Devices" />
+              <p>Internet for all devices</p>
+            </a>
+          </article>
+          <article class="devices">
+            <a>
+              <img src="img/icon-coffee.svg" alt="Coffee" />
+              <p>Boost your productivity</p>
+            </a>
+          </article>
+          <article class="devices">
+            <a>
+              <img src="img/icon-refill.svg" alt="REfill" />
+              <p>Pay as You Go</p>
+            </a>
+          </article>
+        </div>
+      </section>
+    </main>
 
-</body>
+    <footer>
+      <h6>Join us on</h6>
+      <div class="social-media-images">
+        <a href="#"><img src="img/twitter-icon.svg" /></a>
+        <a href="#"><img src="img/facebook-icon.svg" /></a>
+        <a href="#"><img src="img/instagram-icon.svg" /></a>
+      </div>
```

Comment:
Is this a list? 😉

Reply:
Fixed.

<div class="social-media-images">
        <ul class="social-media-images-list">
          <li>
            <a href="#"><img src="img/twitter-icon.svg" /></a>  
          </li>
          <li>
            <a href="#"><img src="img/facebook-icon.svg" /></a>
          </li>
          <li>
            <a href="#"><img src="img/instagram-icon.svg" /></a>
          </li>
        </ul>
      </div>